editor/editor_help.cpp

```diff
@@ -74,6 +75,139 @@ class DocCache : public Resource {
 	void set_classes(const Array &p_classes) { classes = p_classes; }
 };
 
+void EditorHelpImageDownloader::_image_download_completed(int p_status, int p_code, const PackedStringArray &headers, const PackedByteArray &p_data, HTTPRequest *p_rq) {
+	String name = p_rq->get_download_file();
+	if (image_load_requests.has(name)) {
+		Ref<Texture2D> texture;
+		Ref<DirAccess> da = DirAccess::create(DirAccess::ACCESS_FILESYSTEM);
+		if (p_status == HTTPRequest::RESULT_SUCCESS && p_code == 200) {
+			if (da->file_exists(name)) {
+				Ref<Image> img;
+				img.instantiate();
+				Error err = ImageLoader::load_image(name, img);
+				if (err == OK) {
+					texture = ImageTexture::create_from_image(img);
+				} else {
+					da->remove(name); // Delete temp file.
+				}
+			}
+		} else {
+			da->remove(name); // Delete temp file.
+		}
+		if (texture.is_null()) {
+			texture = progress_error;
```

Comment:
How difficult it would be to e.g. display error message and/or the requested URL?

Reply:
Added tooltip with the URL and error message:
<img width="515" alt="Screenshot 2023-07-20 at 23 52 51" src="https://github.com/godotengine/godot/assets/7645683/f20275f4-6e9e-46f8-88b7-37daa8f92737">